src/app/e2e/home_published.spec.js

```diff
@@ -23,23 +29,17 @@ describe('Home page with published data', function homePublishedDataTests() {
         const properties = await driver.findElements(By.css('.dataset-characteristics .property'));
         expect(properties.length).toEqual(2);
 
-        const movieLabel = await driver.findElement(
-            By.css('.dataset-characteristics .property.movie .property_name'),
-        );
-        driver.wait(until.elementTextIs(movieLabel, 'Movie'), DEFAULT_WAIT_TIMEOUT);
+        const movieLabel = '.dataset-characteristics .property.movie .property_name';
```

Comment:
do you see that `.dataset-characteristics .property.movie .property_name` correspond to movieLabel ? me neither.